### ISSUE TITLE
Verdant sprigs only drop from vanilla produce, and when broken specifically(even if right click harvest mod installed)

### ISSUE THREAD

**Issue by Not-February** [Lothrazar/RootsClassic#7]
I have yet to get a single verdant sprig from a mod-added crop from a number of other mods(Roots 2 and Pam's Harvestcraft are the ones I've tried); any sprigs I've gotten have all come from wheat, carrots, potatoes, or beetroots - crops from the base game.

They also do not seem to drop when using mods which allows for harvesting these crops via right-click, at least not with the mod I'm using for it(Harvestables on Curseforge right now, or Harvestcraft when I had it installed in a different setup). They still occasionally drop when the crop is broken when fully grown, but it'd go a long way for the mod's usability if it worked with right-click harvesting mods.

The right click issue also applies to infernal stems from nether wart, if a mod supports right-click harvesting for nether wart, I still only get infernal stems by breaking the fully grown crop.

**Comment by PrinceOfAmber** [comment]
Yes that is correct.  Currently they only drop when the block is physically broken.  

I'll see if theres an easy way to get it hooked into crop harvesting too.
